docs/source/substrafl_doc/substrafl_overview.rst

```diff
@@ -60,22 +60,23 @@ Evaluation Strategy
 
 The evaluation strategy specifies how and when the model is tested. More specifically it defines:
 
-* on which test data the model is tested
-* at which rounds the model is tested
+* The data the model is tested on
+* On which rounds the model is tested
 
 Metric
 ^^^^^^
 
-A metric is a function that computes a performance by comparing labels (ground truth) and model's predictions.
+A metric is a function that computes performance by comparing the model's predictions against labelled data.
 One or several metrics can be added for an :ref:`concept_evaluation_strategy`.
 
 Index Generator
 ^^^^^^^^^^^^^^^
 
-The notion of epochs does not fully apply to the FL setting. Usually we don't want to train on a full epoch on each organization at every round, but on a reduced quantity of data to prevent models from different organizations from diverging too much.
-In a federated setting, at each round, in each organization, the model is trained for ``num_updates`` batches, with each batch containing ``batch_size`` data points.
+The notion of epochs does not fully apply to the FL setting. Training on a full epoch on each organization at every round can often result in divergence, which means a model may miss out on the potential advantages of having different datasets. To best utilize these datasets, we propose training your models on a subset of data within each dataset. This is further explained below (See diagram).
```

Comment:
I would keep the previous formulation as:
- "can often result in divergence" seems weird. It could imply that your local loss does not converge but diverge.
- "we propose training" could imply that we came up with this idea, but actually everybody does that.

I would keep your last sentence though: "This is further explained below (See diagram)."